minimal-launchpad/minimal_ui_index.js

```diff
@@ -69,6 +69,7 @@ async function downloadAndFlash() {
             eraseAll: false,
             compress: true,
         };
+        await esploader.erase_flash();
```

Comment:
can we have this called based on flag (eraseAll) ? if eraseAll is true then call erase_flash() before write_flash()